pkgs/swiftgen/swift2objc/lib/src/generator/generators/class_generator.dart

```diff
@@ -0,0 +1,84 @@
+import 'package:swift2objc/src/ast/_core/shared/referred_type.dart';
+import 'package:swift2objc/src/ast/declarations/compounds/class_declaration.dart';
+import 'package:swift2objc/src/generator/_core/utils.dart';
+
+String generateClass(ClassDeclaration declaration) {
+  final lines = [
```

Comment:
It's clearer to use a multi-line string literal wherever possible. In other cases, like `generateParameters`, building a List/iterable and then calling `join("\n")` is necessary because you have a variable amount of things you're joining. But here you've got a fixed list (other than the methods). So something like this would be clearer (I'd also inline most of the header generator function for clarity):

```Dart
String generateClass(ClassDeclaration declaration) => '''
${_generateClassAnnotations(declatation)}
public class ${declaration.name} ${_generateClassInheritance(declaration)} {
  ${_generateClassWrappedInstance(declaration)}
  ${_generateClassInitializer(declatation)}
  ${_generateClassMethods(declaration).join('\n  ')}
}
''';
```

Reply:
I tried this but sadly the indentation got all messed up. It would make sense to do that when we start feeding the output to a formatter I think  (like I commented on using the nonNulls property)